packages/mdx/mdx-hast-to-jsx.js

```diff
@@ -85,7 +83,9 @@ MDXContent.isMDXComponent = true`
   // Check JSX nodes against imports
   const babelPluginExtractImportNamesInstance = new BabelPluginExtractImportNames()
   const babelPluginExtractExportNamesInstance = new BabelPluginExtractExportNames()
-  const importsAndExports = [importStatements, exportStatements].join('\n')
+  const importsAndExports = []
+    .concat(importStatements, exportStatements)
```

Comment:
was this causing an issue?
Would spread syntax be appropriate here?
```suggestion
  const importsAndExports = [...importStatements, ...exportStatements]
```

Reply:
they used to be two strings, now they’re two arrays.
I’m fine with spread or whatnot but there doesn’t seem to be a well defined baseline on what is and isn’t allowed, so I just did what I was familiar with!

Reply:
Node 8-15 support spread syntax, as well as every major browser (except IE) https://kangax.github.io/compat-table/es6/